Argo/Types/DecodedError.swift

```diff
@@ -0,0 +1,13 @@
+public enum DecodedError: ErrorType {
+  case TypeMismatch(String)
+  case MissingKey(String)
+}
+
+extension DecodedError: CustomStringConvertible {
```

Comment:
Not sure this is really necessary: `String(DecodedError.MissingKey("id"))` should result in `"DecodedError.MissingKey("id")"`.